packages/esm-patient-banner-app/src/banner/patient-banner.test.tsx

```diff
@@ -60,6 +76,7 @@ describe('PatientBanner: ', () => {
     const user = userEvent.setup();
 
     const patientBannerSeachPageProps = { ...testProps, onClick: mockNavigateTo };
+    window.ResizeObserver = ResizeObserverMock;
```

Comment:
Can you try adding the test to check that on decreasing the window size, the count of columns is changing or not.

Reply:
I have catered for this in the Contact details component.